packages/airnode-utilities/src/caching/index.ts

```diff
@@ -0,0 +1,190 @@
+import { existsSync, mkdirSync, readdirSync, readFileSync, rmdirSync, rmSync, statSync, writeFileSync } from 'fs';
+import { join } from 'path';
+import { logger } from '../logging';
+
+export const CACHE_BASE_PATH = `/tmp/airnode-cache`;
+export const CACHE_MAX_FILESYSTEM_AGE_MINUTES = 60;
+export const CACHE_MAX_FILES = 20_000; // (500 (cache size) * 1024 (KB) ) / 4 (KB, fs "cluster size" = 130k
```

Comment:
Nit: It wuld be nice to add the links you found here.

Reply:
Sure thing :)